http_parser.c

```diff
@@ -887,6 +885,12 @@ size_t http_parser_execute (http_parser *parser,
 
       case s_res_status_start:
       {
+        /* See RFC 7230 section 3.3.3, step 1 */
+        if (parser->status_code / 100 == 1 || /* 1xx e.g. Continue */
+            parser->status_code == 204 ||     /* No Content */
+            parser->status_code == 304) {     /* Not Modified */
+            parser->flags |= F_SKIPBODY;
```

Comment:
Indent error, should be two spaces.

Something for a separate commit, since you're here anyway: it might be faster to write `parser->status_code / 100 == 1` as `parser->status_code >= 100 && parser_status_code < 200`.

Long division is moderately expensive and I'm not sure if compilers are smart enough to turn the first into the latter.

Reply:
Thanks, corrected.

Quite surprisingly both clang 6.0.0. and gcc 7.3.0 are that smart actually (at least for x86-64) but I changed this comparison anyway.

clang:
```
        addl    $-100, %eax
        cmpl    $100, %eax
        jb      .LBB0_60
```

gcc:
```
        subl    $100, %eax
        cmpw    $99, %ax
        ja      .L170
```

Reply:
Wouldn't you mind if I mark this conversation resolved?

Reply:
Sorry, of course! I'll resolve it.

Reply:
Thanks!